CHANGES.md

```diff
@@ -2,7 +2,7 @@
 
 ## 9.0.6 (DR9, unreleased)
 
-- No changes yet.
+- Replace `noao.edu` with `noirlab.edu` wherever possible ([PR#150](https://github.com/legacysurvey/legacysurvey/pull/150)).
```

Comment:
Don't forget to document the changes for the raw data page, too.

Reply:
Done.